test/fw/bin/pbs_benchpress

```diff
@@ -63,6 +64,23 @@ from ptl.utils.plugins.ptl_test_tags import PTLTestTags
 from ptl.utils.plugins.ptl_test_data import PTLTestData
 
 
+class PTLNoseConfig(nose.config.Config):
+    def __init__(self, log_format=None, outfile_handler=None, **kw):
+        super().__init__(**kw)
```

Comment:
@ericpershey We need to add `env` in kw here so that it can pick `NOSE_TESTMATCH` environment while calling `nose.config.Config.__init__` else it will use default value of testmatch regex, which will break PTL.

See this: https://github.com/nose-devs/nose/blob/master/nose/config.py#L181 for how it uses `env` in Config class
See this: https://github.com/nose-devs/nose/blob/master/nose/core.py#L108 for how nose.main() passes env if config argument is None

Reply:
Good catch.  Commit ebb19ce should fix this issue.